src/hooks/useSubjectProjects.tsx

```diff
@@ -27,10 +27,7 @@ export function useSubjectProjects() {
         )
         .then((res) => res.data.items),
     {
-      enabled:
-        !!accountHeaders &&
-        (subjects?.length ?? 0) > 0 &&
-        !!subjects?.every((s) => s.projectId),
+      enabled: !!accountHeaders,
```

Comment:
This comment here explains the rationale behind this change:
https://github.com/lifeomic/react-native-sdk/pull/269/files#diff-ca654e5fdb78a75f406342fabca94da850d4e70528a8162257f297b2a9fd89e3R65-R71

Reply:
But the subjects' projectId prop goes into the URL itself.  Re-looking at this, setting a query up to purposefully fail doesn't feel like the right approach.  Along with the loading messages update to come - let's (you or I) make this more intelligent (i.e. to not fetch, but also not get stuck in any endless "loading" states).